app/views/users/show_follow.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <% @users.each do |user| %>
+    <div class="col-md-3">
+      <div style="text-align:center;">
+        <% if user.photo_file_name %>
+          <img class="img-circle" id="profile-photo" style="width:50%;margin-bottom:20px;" src="<%= user.photo_path(:thumb) %>" />
+        <% else %>
+          <img class="img-circle" id="profile-photo" style="width:50%;margin-bottom:20px;" src="https://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(user.email}">
```

Comment:
Wow, this is clever. Could we actually break this into a method called `user.profile_picture` or something, so that we can use Gravatars across the site where they exist? That'll also reduce the complexity here as we'll be able to move the conditional into the model.